probe-rs/src/probe/jlink/mod.rs

```diff
@@ -383,7 +383,7 @@ pub struct JLink {
     speed_khz: u32,
 
     jtag_tms_bits: Vec<bool>,
-    jtag_tdi_bits: Vec<bool>,
+    jtag_tdi_bits: Vec<IoSequenceItem>,
```

Comment:
What does an "Input" value mean for TDI, exactly?

Reply:
That is a good question. Maybe this type is universally where it should not be used.
The change in general is to set up some ground for more work related to SWD protocol. I will try to improve it in regards to JTAG: possibly first by ignoring the in the first step the JTAG protocol.